src/components/Wishlist.tsx

```diff
@@ -24,13 +24,14 @@ function Products() {
 
```

Comment:
Line 23 violates the rules of hooks because it's called conditionally. There's an error in the console.